### PR TITLE
add a draft mentor checkin agenda & ex. goals to mentor role

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

**Comment by bwreid** [comment]
I really like this! The one thing I have been thinking about is that this system requires all of us to be working on one skill implicitly: employee management. We're essentially all taking on the role of what a manager would be thinking about for an individual employee, requiring each of us to get better at that.

I don't think this is necessarily a _problem_ but I wonder if everyone feels good about working on said skill and being somewhat responsible for someone else's development?

**Comment by cap10morgan** [comment]
@bwreid You are right. And I would say that's a fundamental part of the whole "flat team" structure. Not only do you have to "manage yourself," but you also get to "manage" your colleagues to some extent as well. :)

Unshockingly, *I* think doing that makes us a stronger team and each of us stronger software developers (or whatever other career paths we may follow someday).

File: roles/mentor.md [hunk 1]
```diff
@@ -25,6 +25,27 @@ title: Mentor Role
...
 * Be generally available to each other.
 
+## Checkin agenda
+
+1. Any major issues to address first?
+1. What's made you the happiest at work lately?
+1. What's been the most frustrating thing lately?
+1. If you could change one thing about Democracy Works, what would it be and how would you change it?
+1. What would you like to be doing at Democracy Works this time next year?
+1. What would you like to be doing anywhere 5 years from now?
```
Comment (7maples) [inline roles/mentor.md]: Some of these don't make sense on a monthly basis, but may be more appropriate annually.

**Comment by 7maples** [comment]
I think we have two discussions: what should be in a checkin, and how do we best manage our team? and they're getting muddled together.

Recently I saw a good article on suggestions for one-on-one topics, sharing that [here](http://jvns.ca/blog/2015/03/06/1-1-topic-ideas/). 

I would feel more comfortable talking about the other issue in a checkin, for which these sorts of sensitive topics are better suited and demonstrative of their need, when those recommence.

File: roles/mentor.md (hunk 1 above)
Comment (cap10morgan) [inline roles/mentor.md]: Yep, we should definitely put a recommended frequency on some of these.

**Comment by cap10morgan** [comment]
I'd also like to propose dropping the "mentor" label on this role and making it just about dev team checkins. Objections?

**Comment by ericnormand** [comment]
LGTM

**Pull request merged**